release-tool.ps1

```diff
@@ -0,0 +1,456 @@
+Write-Host "Copyright (C) 2021 KeePassXC Team <https://keepassxc.org/>`n" -ForegroundColor Green
+
+# Prepend extra PATH locations as specified
+if ($ExtraPath) {
+    $env:Path = "$ExtraPath;$env:Path"
+}
+
+# Resolve absolute directory for paths
+$SourceDir = (Resolve-Path $SourceDir).Path
+
+# Check format of -Version
+if ($Version -notmatch "^\d+\.\d+\.\d+$") {
+    throw "Invalid format for -Version input"
+}
+
+# Check platform
+if (-not $IsWindows) {
+    throw "The powershell release tool is not available for Linux or macOS at this time."
+}
+
+if ($Merge) {
+    Test-RequiredPrograms
+
+    # Change to SourceDir
+    Set-Location "$SourceDir"
+
+    Test-VersionInFiles
+    Test-WorkingTreeClean
+
+    if (-not $SourceBranch.Length) {
+        $SourceBranch = $(git branch --show-current)
+    }
+
+    if ($SourceBranch -notmatch "^release/.*|develop$") {
+        throw "Must be on a Release or Develop branch to continue with merge."
+    }
+
+    # Update translation files
+    Write-Host "Updating source translation file..."
+    Invoke-Cmd "lupdate" "-no-ui-lines -disable-heuristic similartext -locations none", `
+        "-no-obsolete ./src -ts share/translations/keepassxc_en.ts"
+
+    Write-Host "Pulling updated translations from Transifex..."
+    Invoke-Cmd "tx" "pull -af --minimum-perc=60 --parallel -r keepassxc.share-translations-keepassxc-en-ts--develop"
+
+    # Only commit if there are changes
+    $(git diff-index --quiet HEAD --)
+    if ($LASTEXITCODE) {
+        Write-Host "Committing translation updates..."
+        Invoke-Cmd "git" "add -A ./share/translations/" -quiet
+        Invoke-Cmd "git" "commit -m `"Update translations`"" -quiet
+    }
+
+    # Read the version release notes from CHANGELOG
+    $Changelog = ""
+    $ReadLine = $false
+    Get-Content "CHANGELOG.md" | ForEach-Object {
+        if ($ReadLine) {
+            if ($_ -match "^## ") {
+                $ReadLine = $false
+            } else {
+                $Changelog += $_ + "`n"
+            }
+        } elseif ($_ -match "$Version \(\d{4}-\d{2}-\d{2}\)") {
+            $ReadLine = $true
+        }
+    }
+
+    Write-Host "Checking out target branch '$TargetBranch'..."
+    Invoke-Cmd "git" "checkout `"$TargetBranch`"" -quiet
+
+    Write-Host "Merging '$SourceBranch' into '$TargetBranch'..."
+    Invoke-Cmd "git" "merge `"$SourceBranch`" --no-ff -m `"Release $Version`" -m `"$Changelog`" `"$SourceBranch`" -S" -quiet
+
+    Write-Host "Creating tag for '$Version'..."
+    Invoke-Cmd "git" "tag -a `"$Version`" -m `"Release $Version`" -m `"$Changelog`" -s" -quiet
+
+    Write-Host "All done!"
+    Write-Host "Please merge the release branch back into the develop branch now and then push your changes."
+    Write-Host "Don't forget to also push the tags using 'git push --tags'."
+}
+elseif ($Build) {
+    $OutDir = (Resolve-Path $OutDir).Path
+    $BuildDir = "$OutDir\build-release"
+    $Vcpkg = (Resolve-Path $Vcpkg).Path
```

Comment:
Here you don't use `$( ... )`.

Reply:
I am not calling a native command here, I am wrapping the return value from the Resolve-Path call so I can get the .Path attribute from the returned object.